sources/views/contacts.js

```diff
@@ -1,43 +1,83 @@
 import { JetView } from "webix-jet";
 import { contacts } from "models/contacts";
 
-import Form from "views/form";
-
 export default class Contacts extends JetView {
 	config() {
-		const list = {
-			padding: {
-				right: 10
-			},
-			rows: [
+		const _ = this.app.getService("locale")._;
+		return {
+			cols: [
 				{
-					type: "header",
-					template: "Contacts",
-					height: 50,
-					css: "webix_dark"
+					padding: {
+						right: 10
+					},
+					rows: [
+						{
+							type: "header",
+							template: _("Contacts"),
+							height: 50,
+							css: "webix_dark"
+						},
+						{
+							view: "list",
+							localId: "contactsList",
+							autowidth: true,
+							scroll: false,
+							select: true,
+							type: {
+								height: 62
+							},
+							template: `
+								<b>${_("Name")}: </b>#Name# 
+								<span class="webix_icon wxi-close remove_icon" style="float: right"></span> 
+								<br> 
+								<b>${_("Email")}: </b>#Email# `,
+							onClick: {
+								"wxi-close": (e, id) => this.deleteItem(id)
+							},
+						},
+						{
+							view: "button",
+							label: _("Add"),
+							align: "right",
+							autoWidth: true,
+							css: "webix_primary",
+							click: () => this.addItem()
+						},
+
+					]
 				},
-				{
-					view: "list",
-					localId: "contactsList",
-					autowidth: true,
-					scroll: false,
-					select: true,
-					template: "#Name# <br> #Email# <span class='webix_icon wxi-close right'></span>"
-				}
+				{ $subview: true }
 			]
 		};
+	}
+	init(view, url) {
+		this.list = this.$$("contactsList");
+		this.list.sync(contacts);
+		this.list.attachEvent("onItemClick", id => {
+			this.show(`./form?id=${id}`);
+		});
 
-		const ui = {
-			cols: [
-				list,
-				Form
-			]
-		};
+		if (url.length <= 1) {
+			this.show(`./form?id=${this.list.getFirstId()}`);
+		}
+	}
+
+	urlChange(view, url) {
+		const id = url.length <= 1 ? (url[0].params.id || this.list.getFirstId()) : (url[1].params.id);
+		id ? this.list.select(id) : this.list.unselectAll();
+	}
 
-		return ui;
+	addItem() {
+		this.show(`./form`);
```

Comment:
adding though form is possible, but as a user I cannot understand that happens, because nothing happens in UI except of  url change. you can: 

- (better) either add an empty contact and immediately open it for editing in the form
- fill in the form with some fake values like "New contact"

Reply:
why first way is better. in this case, you clearly separate the functionality: "Add" button is responsible for adding, and  "Save" is only for updating. moreover, for user it is enough to press only "add" to really add the contact (although it is empty at the moment)

Reply:
and - during adding, do not forget to show the newly added item

Reply:
If it possible, I want to leave both ways to add new users (via add button from the list and via save button from the form). Because the user may want to simply enter their data into the form and click the add button. If this is not the right decision, I’ll remove it.